### ISSUE TITLE
Error when using minified version.

### ISSUE THREAD

**Issue by ryanwiesjahn** [projectweekend/angular-skycons#8]
When I use the minified version, I am getting this console error:
```
TypeError: a.drawing is not a function
    at Object.Skycons.draw (http://localhost/myapp/bower_components/angular-skycons/angular-skycons.min.js:1:7570)
    at Object.Skycons.add (http://localhost/myapp/bower_components/angular-skycons/angular-skycons.min.js:1:6998)
    at link (http://localhost/myapp/bower_components/angular-skycons/angular-skycons.min.js:1:8244)
    at http://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:70:370
    at $ (http://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:70:426)
    at v (http://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:59:401)
    at g (http://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:52:9)
    at g (http://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:52:26)
    at g (http://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:52:26)
    at g (http://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:52:26) <skycon icon="weather.skycon.icon" class="ng-isolate-scope">
```

Order of my scripts:
```
<script src="//ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js"></script>
<script src="//code.angularjs.org/1.3.8/angular-animate.min.js"></script>
<script src="assets/js/ui-router.min.js"></script>
<script src="bower_components/angular-skycons/angular-skycons.min.js"></script>
```

It works fine when not using minified version.

**Comment by projectweekend** [comment]
Thanks for catching this! Not sure what happened with my Grunt task. Just pushed up a fix and retagged so it is available on Bower as v2.0.2. Check it out and let me know if you still have problems.

**Comment by projectweekend** [comment]
@ryanwiesjahn I pushed an updated version for Bower last week. Let me know if you have any issues with that version. I am closing the issue. Thanks!
